mnemonic-core/src/main/java/org/apache/mnemonic/VolatileMemAllocator.java

```diff
@@ -29,7 +29,8 @@
  * 
  *
  */
-public class VolatileMemAllocator extends RestorableAllocator<VolatileMemAllocator> {
+public class VolatileMemAllocator extends RestorableAllocator<VolatileMemAllocator> 
+  implements AddressTranslator {
```

Comment:
Please remove this change because it parent class already implemented the interface of AddressTranslator.